### ISSUE TITLE
Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (11 - Normative)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#453]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

3.	Outcomes are normative. Should guidelines, methods, critical errors, and outcome ratings be normative or informative, and why?
Guidelines should be normative, and normative text should also indicate that conformance with the guidelines is determined through aggregate scores for outcomes. The outcome ratings and critical errors also need to be normative as these are used to determine overall conformance with the standard. Methods are technology and situation-specific, much like the current WCAG techniques, and these should be informative.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
